src/Website/Views/ContactUs/ContactUs.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿
+@{
+    ViewBag.Title = "Contact Us";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<article>
+    <header>
+        <hgroup>
+            <h1>The Westman Alers</h1>
+        </hgroup>
+    </header>
+    <section>
+        <h2>Who We Are</h2>
+        <p>Description of the club, its activities and goals go here.</p>
+    </section>
+    <section>
+        <h2>How To Contact Us</h2>
+        <p>Email address goes here.</p>
+    </section>
+</article>
+
```

Comment:
Since we're designing this site to be re-usable (in theory), should the about/contact info be stored in the database and configurable by the admins?